buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -346,6 +409,7 @@ private void configure()  {
             config.put("cluster.initial_master_nodes", "[" + nodeName + "]");
         }
         try {
+            Files.delete(configFile);
```

Comment:
Why do we need to delete the config file when it is written below this?

Reply:
I removed it, it's an oversight.